### ISSUE TITLE
Is it possible to create a saved search with an XML or JSON search?

### ISSUE THREAD

**Issue by bytesandwich** [splunk/terraform-provider-splunk#42]
Suppose I've taken (many) saved searches out of some existing splunk, put them through the `search/parser` endpoint, run a translation against these to change them all, and now I want to put them into a new splunk instance using the terraform saved_search object. Is this possible?

Can I somehow just put the json or xml into the saved search object or can I somehow get the search string for a mutated json search object?

**Comment by anushjay** [comment]
@bytesandwich 
I don't think this is possible at this time. The saved_searches endpoint doesn't take a json blob of the whole search as input.
I see that you asked the question in splunk/splunk-sdk-python repo as well so may be we can wait to see if they have a postivie response.

You can see if the configs resource in the terraform provider can help you solve the problem. 🤔
